client/client.go

```diff
@@ -148,5 +152,15 @@ func Cred(configFilePath string) (*Key, error) {
 		return nil, fmt.Errorf("invalid public key type: %T", publicKey)
 	}
 
+	switch pub := k.publicKey.(type) {
+	case *rsa.PublicKey:
+		if pub.Size() < 256 {
+			return nil, fmt.Errorf("RSA modulus size is less than 2048 bits: %v", pub.Size())
```

Comment:
pub.Size() seems to be in bytes. So error msg will have both bits and bytes which will be confusing.

Reply:
Good catch. (I updated to log the pub key size in bits, as it is more common to refer to modulus size in bits than byte.)